openmrs/src/main/java/org/motechproject/openmrs/tasks/builder/ActionBuilder.java

```diff
@@ -156,16 +158,32 @@ private ActionEventRequest buildCreateProgramEnrollmentAction(String configName)
                 .createActionEventRequest();
     }
 
+    private ActionEventRequest buildUpdateProgramEnrollmentAction(String configName) {
+        SortedSet<ActionParameterRequest> parameters = new TreeSet<>();
+        int order = 0;
+        String serviceMethod = "updateProgramEnrollment";
+
+        parameters.add(prepareParameter(Keys.CONFIG_NAME, DisplayNames.CONFIG_NAME, configName, false, true, order++));
+        parameters.addAll(prepareProgramEnrollmentParameters(order));
+        order = parameters.size();
+        parameters.add(prepareParameter(Keys.PROGRAM_ATTRIBUTES, DisplayNames.PROGRAM_ATTRIBUTES, MAP, false, order));
```

Comment:
it would be better to add this param in the prepareProgramEnrollmentParameters method. Maybe add a flag to exclude/include this param depends on chosen action